ftw/book/footnote/footnote-plugin/editor_plugin.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * TinyMCE footnote plugin
+ */
+
+(function() {
+  tinyMCE.addI18n('en.footnote',{
+    title : 'Footnote'
+  });
+
+  tinyMCE.addI18n('de.footnote',{
+    title : 'Fussnote'
+  });
+
+
+  tinymce.create('tinymce.plugins.Footnote', {
+    init : function(ed, url) {
+      this.editor = ed;
+
+      // Register commands
+      ed.addCommand('mceFootnote', function() {
+        var se = ed.selection;
+
+        ed.windowManager.open({
+          file : url + '/footnote.htm',
+          width : 600 + parseInt(
+              ed.getLang('advanced.anchor_delta_width', 0), 10),
+          height : 190 + parseInt(
+              ed.getLang('advanced.anchor_delta_height', 0), 10),
+          inline : 1
+        }, {
+          plugin_url : url
+        });
+      });
+
+      // Register buttons
+      ed.addButton('tinymce_footnote', {
+        title : 'footnote.title',
+        cmd : 'mceFootnote',
+        img : ''
+      });
+
+      ed.onNodeChange.add(function(ed, cm, n, co) {
+        cm.setDisabled('tinymce_footnote', co && n.nodeName != 'span');
+        cm.setActive('tinymce_footnote', n.nodeName == 'span' && !n.title);
```

Comment:
Yes it should be `data-footnote`. But on further inspection i realized that this is not relevant because TinyMCE gives the tag name in upper-case. :stuck_out_tongue_winking_eye: 
I refactored this function as well as the equivalent in the keynote plugin in https://github.com/4teamwork/ftw.book/commit/c2a912164dbcbf27d7385ab4dc2082150d20f47b.